src/main/java/com/iridium/iridiumskyblock/managers/IslandManager.java

```diff
@@ -202,6 +203,13 @@ private void deleteIslandBlocks(Island island, World world, int y, CompletableFu
 
     @Override
     public void deleteTeam(Island island, User user) {
+        final IslandDeleteEvent islandDeleteEvent = new IslandDeleteEvent(island, user);
+        Bukkit.getPluginManager().callEvent(islandDeleteEvent);
+
+        if (islandDeleteEvent.isCancelled()) {
+            return;
```

Comment:
might be worth modifying this method to return a boolean.

In Teams, https://github.com/Iridium-Development/IridiumTeams/blob/master/src/main/java/com/iridium/iridiumteams/commands/DeleteCommand.java#L62 we will still send the deleted message. if this returns a boolean we can optionally send that.

Would require changing the https://github.com/Iridium-Development/IridiumTeams/blob/master/src/main/java/com/iridium/iridiumteams/managers/TeamManager.java#L89 Interface.

Feel free to open a PR in teams yourself, or ill do it myself over the weekend